package.json

```diff
@@ -6,7 +6,7 @@
   "author": "The Gaia Team <dev-gaia@lists.mozilla.org>",
   "engineStrict": "true",
   "engines": {
-    "node": ">=0.10",
+    "node": ">=4.0",
```

Comment:
There would be a conflict reported in npm commands, so yeah - we probably should bump the gaia requested version and engines as well.